WalletWasabi.Packager/Program.cs

```diff
@@ -499,16 +500,17 @@ private static async Task PublishAsync()
 
 				commands = new[]
 				{
-						"cd ~",
-						"sudo umount /mnt/c",
-						"sudo mount -t drvfs C: /mnt/c -o metadata",
-						$"cd {linuxPath}",
-						$"sudo find {Tools.LinuxPath(newFolderRelativePath)} -type f -exec chmod 644 {{}} \\;",
-						$"sudo find {Tools.LinuxPath(newFolderRelativePath)} {chmodExecutablesArgs}",
-						$"sudo chmod -R 0775 {Tools.LinuxPath(debianFolderRelativePath)}",
-						$"sudo chmod -R 0644 {debDestopFileLinuxPath}",
-						$"dpkg --build {Tools.LinuxPath(debFolderRelativePath)} $(pwd)"
-					};
+					$"cd /",
+					$"sudo umount -l /mnt/c",
```

Comment:
This block is duplicated and in the one above the drive letter is not harcoded. I think we can avoid this duplication and extract the common parts.

Reply:
Yes, I have refactored it a bit. 

We can refactor it much more but one step at a time I guess.